[CI] InternalDistributionBwcSetupPluginFuncTest class failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #3479 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/xu36cavvwr6rs)
- [elasticsearch-periodic-platform-support #3479 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/xu36cavvwr6rs)
- [elasticsearch-periodic-platform-support #3479 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/xu36cavvwr6rs)
- [elasticsearch-periodic-platform-support #3479 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/xu36cavvwr6rs)
- [elasticsearch-periodic-platform-support #3475 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/udpr7l6eetfp2)
- [elasticsearch-periodic-platform-support #3475 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/udpr7l6eetfp2)
- [elasticsearch-periodic-platform-support #3475 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/udpr7l6eetfp2)
- [elasticsearch-periodic-platform-support #3475 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/udpr7l6eetfp2)
- [elasticsearch-periodic-platform-support #3471 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/b4y3qldz3lx5a)
- [elasticsearch-periodic-platform-support #3471 / windows-2022_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/b4y3qldz3lx5a)

**Reproduction Line:**
```
null
```

**Applicable branches:**
7.17

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('builds%20distribution%20from%20branches%20via%20archives%20assemble'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.gradle.internal.InternalDistributionBwcSetupPluginFuncTest'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
org.gradle.testkit.runner.UnexpectedBuildFailure: Unexpected build execution failure in C:\Users\BUILDK~1\AppData\Local\Temp\junit13989225688115742975\cloned with arguments [:distribution:bwc:minor:buildBwcLinuxAarch64Tar, :distribution:bwc:minor:buildBwcOssLinuxAarch64Tar, -DtestRemoteRepo=C:\Users\BUILDK~1\AppData\Local\Temp\junit11623305603273656106\remote\.git, -Dbwc.remote=origin, -Dbwc.dist.version=7.12.0-SNAPSHOT, -Dtest.external=true]

Output:
> Task :buildSrc:compileJava NO-SOURCE
> Task :buildSrc:compileGroovy NO-SOURCE
> Task :buildSrc:processResources
> Task :buildSrc:classes
> Task :buildSrc:jar
Directory 'C:\Users\buildkite\.java\java11' (environment variable 'JAVA11_HOME') used for java installations does not exist
=======================================
Elasticsearch Build Hamster says Hello!
  Gradle Version        : 8.9
  OS Info               : Windows Server 2022 10.0 (amd64)
  JDK Version           : 17.0.2+8-86 (Oracle)
  JAVA_HOME             : C:\Users\buildkite\.java\openjdk17
  Random Testing Seed   : 2AD154DC1447511C
  In FIPS 140 mode      : false
=======================================
> Task :distribution:bwc:minor:createClone
> Task :distribution:bwc:minor:findRemote
> Task :distribution:bwc:minor:addRemote SKIPPED
> Task :distribution:bwc:minor:fetchLatest

> Task :distribution:bwc:minor:checkoutBwcBranch
Performing checkout of origin/7.x...
Checkout hash for :distribution:bwc:minor is 3e766eed1012758552923e83c124436c809a3818

> Task :distribution:bwc:minor:buildBwcLinuxAarch64Tar FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':distribution:bwc:minor:buildBwcLinuxAarch64Tar'.
> JAVA11_HOME required

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 12s
7 actionable tasks: 7 executed

```

**Issue Reasons:**
- [7.17] 3 consecutive failures in step windows-2022_checkpart1_platform-support-windows
- [7.17] 3 failures in class supports #platform aarch distributions (30.0% fail rate in 10 executions)
- [7.17] 3 failures in class builds distribution from branches via archives extractedAssemble (30.0% fail rate in 10 executions)
- [7.17] 3 failures in class builds distribution from branches via archives assemble (30.0% fail rate in 10 executions)
- [7.17] 3 failures in class org.elasticsearch.gradle.internal.InternalDistributionBwcSetupPluginFuncTest (30.0% fail rate in 10 executions)
- [7.17] 3 failures in step windows-2022_checkpart1_platform-support-windows (75.0% fail rate in 4 executions)
- [7.17] 3 failures in pipeline elasticsearch-periodic-platform-support (75.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/build-tools-internal/src/integTest/groovy/org/elasticsearch/gradle/internal/InternalDistributionBwcSetupPluginFuncTest.groovy
+++ b/build-tools-internal/src/integTest/groovy/org/elasticsearch/gradle/internal/InternalDistributionBwcSetupPluginFuncTest.groovy
@@ -9,16 +9,18 @@
 package org.elasticsearch.gradle.internal
 
 import org.elasticsearch.gradle.Architecture
+import org.elasticsearch.gradle.OS
 import org.elasticsearch.gradle.fixtures.AbstractGitAwareGradleFuncTest
 import org.gradle.testkit.runner.TaskOutcome
 import spock.lang.IgnoreIf
 import spock.lang.Unroll
 
 /*
- * Test is ignored on ARM since this test case tests the ability to build certain older BWC branches that we don't support on ARM
+ * Test is ignored on ARM since this test case tests the ability to build certain older BWC branches that we don't support on ARM.
+ * We also ignore on Windows since we don't run BWC tests on that platform anyway.
  */
 
-@IgnoreIf({ Architecture.current() == Architecture.AARCH64 })
+@IgnoreIf({ Architecture.current() == Architecture.AARCH64  || OS.current() == OS.WINDOWS })
 class InternalDistributionBwcSetupPluginFuncTest extends AbstractGitAwareGradleFuncTest {
 
     def setup() {

